app/controllers/about/feedback_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module About
+  class FeedbackController < ApplicationController
+    def index
+      @feedback = Feedback.new
+    end
+
+    def create
+      @feedback = Feedback.new feedback_params
+      if @feedback.valid?
+        submit_feedback
+        redirect_to applications_path, notice: I18n.t('feedback.submitted.success')
+      else
+        render 'index'
+      end
+    end
+
+    private
+
+    def submit_feedback
+      FeedbackMailer.notify(
+        user_email: feedback_params[:user_email],
+        user_rating: feedback_params[:user_rating],
+        user_feedback: feedback_params[:user_feedback],
+        application_env: application_environment
+      ).deliver_later!
```

Comment:
```
      FeedbackMailer.notify(
        application_env: application_environment
        **feedback_params
      ).deliver_later!
```

is the same. :)

Reply:
This throws an exception as expecting the params - there are default params so that might be the case